src/testsuite/arith128_test_i64.c

```diff
@@ -948,6 +948,195 @@ test_rldi (void)
 }
 #undef __DEBUG_PRINT__
 
+static unsigned long long test_ui64[] =
+    {
+	0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15
+    };
+
+int
+test_lvgudx (void)
+{
+  vui64_t i1, i2, e;
+  vui64_t j, j0, j1;
+  int rc = 0;
+
+  printf ("\ntest_Load Vector with gather Doubleword\n");
+
+  e =  (vui64_t) CONST_VINT64_DW ( 0, -1 );
+  j0 = vec_vlxsdx (0, test_ui64);
+  j = vec_permdi (j0, e, 1);
+
+  rc += check_vuint128x ("vec_lxsdx 1:", (vui128_t) j, (vui128_t) e);
+
+  e =  (vui64_t) CONST_VINT64_DW ( 1, -1 );
+  j0 = vec_vlxsdx (8, test_ui64);
+  j = vec_permdi (j0, e, 1);
+
+  rc += check_vuint128x ("vec_lxsdx 2:", (vui128_t) j, (vui128_t) e);
+
+  e =  (vui64_t) CONST_VINT64_DW ( 15, -1 );
+  j0 = vec_vlxsdx (120, test_ui64);
+  j = vec_permdi (j0, e, 1);
+
+  rc += check_vuint128x ("vec_lxsdx 3:", (vui128_t) j, (vui128_t) e);
+
+  i1 = (vui64_t) { 8, 120 };
+  e =  (vui64_t) CONST_VINT64_DW ( 1, -1 );
+  j0 = vec_vlxsdx (i1[0], test_ui64);
+  j = vec_permdi (j0, e, 1);
+
+  rc += check_vuint128x ("vec_lxsdx 4:", (vui128_t) j, (vui128_t) e);
+
+  i1 = (vui64_t) { 8, 120 };
+  e =  (vui64_t) CONST_VINT64_DW ( 15, -1 );
+  j1 = vec_vlxsdx (i1[1], test_ui64);
+  j = vec_permdi (j1, e, 1);
+
+  rc += check_vuint128x ("vec_lxsdx 5:", (vui128_t) j, (vui128_t) e);
+
+  e =  (vui64_t) { 1, 15 };
+#if (__BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__)
+  j = vec_permdi (j1, j0, 0);
+#else
+  j = vec_permdi (j0, j1, 0);
+#endif
+
+  rc += check_vuint128x ("vec_lxsdx2 :", (vui128_t) j, (vui128_t) e);
```

Comment:
maybe add a comment that this test depends on the results of the previous two tests.

Reply:
// This test depends on the merge of scalars from lxsdx 4/5